src/basis/l10n.js

```diff
@@ -24,7 +52,7 @@
     dictionary = resolveDictionary(url);
     autoFetchDictionaryResource = true;
 
-    return dictionary.update(basis.resource.extensions['.json'](content, url));
```

Comment:
because it's not processed for now
`'.json': function processJsonResourceContent(content){...}`